README.md

```diff
@@ -10,58 +10,58 @@ Below is jp-en compatible table include a bit joke. Patches welcome!
 
 <table>
     <tr>
-        <th>view</th>
-        <th>code example</th>
-        <th>japanese</th>
-        <th>meaning</th>
+        <th>View</th>
+        <th>Code Example</th>
+        <th>Japanese</th>
+        <th>Meaning</th>
     </tr>
     <tr>
         <td><img src="https://raw.githubusercontent.com/oti/slack-reaction-decomoji/master/dist/ahhai.png"></td>
         <td><code>:ahhai:</code></td>
         <td>あっハイ</td>
-        <td>"well."</td>
+        <td>"Ah, yeah."</td>
```

Comment:
How about to use `Oh yah.` ?
http://www.urbandictionary.com/define.php?term=OH+YAH